flink-connectors/flink-connector-hive/src/main/resources/META-INF/NOTICE

```diff
@@ -0,0 +1,41 @@
+flink-connector-hive
+Copyright 2014-2019 The Apache Software Foundation
+
+This product includes software developed at
+The Apache Software Foundation (http://www.apache.org/).
+
+This project bundles the following dependencies under the Apache Software License 2.0. (http://www.apache.org/licenses/LICENSE-2.0.txt)
+
+- org.apache.hadoop:hadoop-common:2.7.2
+- commons-io:commons-io:2.4
+- commons-logging:commons-logging:1.1.3
+- commons-configuration:commons-configuration:1.7
+- commons-beanutils:commons-beanutils:1.8.3
+- org.apache.hadoop:hadoop-auth:2.7.2
+- org.apache.hadoop:hadoop-mapreduce-client-core:2.7.2
+- org.apache.hive:hive-metastore:2.3.4
+- org.apache.hive:hive-serde:2.3.4
+- org.apache.hive:hive-common:2.3.4
+- org.apache.hive:hive-storage-api:2.4.0
+- org.apache.hive:hive-service-rpc:2.3.4
+- com.jolbox:bonecp:0.8.0.RELEASE
+- org.apache.derby:derby:10.10.2.0
+- org.datanucleus:datanucleus-api-jdo:4.2.4
+- org.datanucleus:datanucleus-core:4.1.17
+- org.datanucleus:datanucleus-rdbms:4.1.19
+- commons-io:commons-pool:2.4
```

Comment:
when I built it locally `commons-pool:commons-pool:jar:1.5.4` was bundled

Reply:
copy-paste error. fixed